test/common.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"bytes"
+	"embed"
+	cmpact "encoding/json"
+	"github.com/goccy/go-json"
+	"fmt"
+	manifestsdk "github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/oliveagle/jsonpath"
+	"github.com/pkg/errors"
+	credmodel "github.com/tbd54566975/ssi-service/internal/credential"
+	"io"
+	"net/http"
+)
+
+const (
+	endpoint = "http://localhost:8080/"
+	version  = "v1/"
+)
+
+var (
+	//go:embed testdata
+	testVectors embed.FS
+)
+
+func compactJSONOutput(json string) string {
```

Comment:
Curious what's the reason for using the `encoding/json` vs the other json function?

Reply:
good call - we should be using `go-json` everywhere